lib/Models/ScheduledTaskSubscriber.php

```diff
@@ -86,6 +86,19 @@ public static function listingQuery($data) {
       ->select('subscribers.id', 'subscriberId')
       ->select('subscribers.last_name', 'lastName')
       ->select('subscribers.first_name', 'firstName');
+    if (isset($data['search'])) {
+      $search = trim($data['search']);
+      $search = str_replace(['\\', '%', '_'], ['\\\\', '\\%', '\\_'], $search); // escape for 'LIKE'
+      if (strlen($search) === 0) {
+        return $query;
+      }
+
+      return $query->whereRaw(
+        '(`subscribers.email` LIKE ? OR `subscribers.first_name` LIKE ? OR `subscribers.last_name` LIKE ?)',
+        ['%' . $search . '%', '%' . $search . '%', '%' . $search . '%']
```

Comment:
I would prefer to do `$search = '%' . $search . '%'` before then just use `$search` here, I think it will be cleaner that way.